dlp/api/Snippets.Tests/DeidentifyFreeTextWithFpeUsingSurrogateTests.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+using Google.Cloud.Dlp.V2;
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class DeidentifyFreeTextWithFpeUsingSurrogateTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+        public DeidentifyFreeTextWithFpeUsingSurrogateTests(DlpTestFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void TestDeidentifyFreeText()
+        {
+            var text = "My phone number is 4359916732.";
+            var unwrappedKey = "YWJjZGVmZ2hpamtsbW5vcB==";
```

Comment:
@annadata is this in any way sensitive and should be better off in an env variable?

Reply:
This is not sensitive information. Moving this to Env vars will make it look elegant. Since this is in a test case, it is probably ok. In some of the languages, we have used a key generated on the fly. The advantage with this approach is we can have an assert on expected output.

> "My phone number is PHONE_TOKEN(10):9617256398."